tests/labhub_test.py

```diff
@@ -82,31 +82,41 @@ def test_create_issue_cmd(self):
         plugins.labhub.GitHubToken = create_autospec(IGitt.GitHub.GitHubToken)
         plugins.labhub.GitLabPrivateToken = create_autospec(IGitt.GitLab.GitLabPrivateToken)
 
-        labhub, testbot = plugin_testbot(plugins.labhub.LabHub, logging.ERROR,
-                                         {'BACKEND': 'text'})
+        labhub, testbot_private = plugin_testbot(plugins.labhub.LabHub, logging.ERROR,
+                                         {'BACKEND': 'text',
```

Comment:
this alignment is bad. move all args to the next line, indented by four spaces, and add indents from there as appropriate.

Reply:
Fixed it!